### PR TITLE
Listening to multiple queues

### PR DESCRIPTION
**Pull request by guilhermef**
Listens to multiple queues and routes the message to the desired app.

### CODE REVIEW

File: extensions/apns_message_handler_test.go [hunk 1]
```diff
@@ -357,38 +357,47 @@ var _ = Describe("APNS Message Handler", func() {
...
-					&stopChannel,
-					mockKafkaConsumerClient,
-				)
-				Expect(err).NotTo(HaveOccurred())
-				Expect(func() { go handler.HandleMessages(queue.MessagesChannel()) }).ShouldNot(Panic())
-				time.Sleep(50 * time.Millisecond)
-				Expect(handler.run).To(BeTrue())
-			})
-		})
+		// Describe("Handle Messages", func() {
```
Comment (cscatolini) [inline extensions/apns_message_handler_test.go]: why is this test commented?

File: extensions/gcm_message_handler_test.go [hunk 2]
```diff
@@ -346,31 +373,37 @@ var _ = Describe("GCM Message Handler", func() {
...
-					handler.Config, logger,
-					&stopChannel, mockKafkaConsumerClient,
-				)
-				Expect(err).NotTo(HaveOccurred())
-				Expect(func() { go handler.HandleMessages(queue.MessagesChannel()) }).ShouldNot(Panic())
-				time.Sleep(time.Millisecond)
-				Expect(handler.run).To(BeTrue())
-			})
-		})
+		// Describe("Handle Messages", func() {
```
Comment (cscatolini) [inline extensions/gcm_message_handler_test.go]: why is this test also commented?

File: extensions/token_pg.go [hunk 3]
```diff
@@ -73,13 +71,13 @@ func (t *TokenPG) configure(db interfaces.DB) error {
...
 // HandleToken handles an invalid token
-func (t *TokenPG) HandleToken(token string) error {
+func (t *TokenPG) HandleToken(token string, game string, platform string) error {
 	l := t.Logger.WithFields(logrus.Fields{
 		"method": "HandleToken",
 		"token":  token,
 	})
 	l.Debug("deleting token")
-	query := fmt.Sprintf("DELETE FROM %s WHERE token = ?0;", t.tableName)
+	query := fmt.Sprintf("DELETE FROM %s WHERE token = ?0;", game+"_"+platform)
```
Comment (cscatolini) [inline extensions/token_pg.go]: I am not sure all games respect this rule =(

Comment (felipejfc) [reply extensions/token_pg.go]: omg, we need to be sure @guilhermef

**Pull request merged**